www/test/hack/bootstrap-web-e2e.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script to test web ui e2e tests using protractor works as expected.
```

Comment:
also wondering whether this should be put under $KUBE_ROOT/hack (keep all the hacks together?) rather than creating yet another hack directory.